Source: timothy@redaelli.eu
Upstream: no
Reason: we need an error code if a test is failed

--- lsof_4.82_src/tests/Makefile.orig	2010-01-14 10:47:09.959781986 +0000
+++ lsof_4.82_src/tests/Makefile	2010-01-14 10:52:31.355542398 +0000
@@ -27,7 +27,7 @@
 	  exit 1 ;\
 	fi
 	@rm -f config.LT*
-	-@err=0; \
+	@err=0; \
 	echo ""; \
 	echo "Basic test:"; \
 	./${BASTST}; \
@@ -46,6 +46,7 @@
 	  echo "Failed tests: $$err"; \
 	  echo ""; \
 	  echo "See 00FAQ and 00TEST for more information."; \
+	  exit $$err; \
 	else \
 	  echo "All standard tests succeeded."; \
 	  echo ""; \
@@ -112,7 +113,7 @@
 
 opt:	${CKTSTDB} ${OPTTST} FRC
 	@rm -f config.LT*
-	-@err=0; \
+	@err=0; \
 	echo ""; \
 	echo "Optional tests:"; \
 	for i in ${OPTTST}; do \
@@ -123,6 +124,7 @@
 	done; \
 	if [ $$err -ne 0 ]; then \
 	  echo "Failed tests: $$err"; \
+	  exit $$err; \
 	else \
 	  echo "All optional tests succeeded."; \
 	fi; \
